web/src/App.js

```diff
@@ -33,19 +30,36 @@ class App extends Component {
   handleUsername = username => {
     this.setState({
       username: username,
-      tweets: []
+      tweets: [],
+      error: false
     });
-    service.getInfo(username).then(this.load);
+    service.getInfo(username).then(this.load).catch(this.loadError);
   };
 
   load = info => {
+    if (!info && !info.model) {
```

Comment:
The model does not come filled when there's an error, like the one we are handling here. Model comes undefined, so no user data retrieved.